WooCommerce/Classes/ViewRelated/ReusableViews/SwiftUI Components/AttributedText.swift

```diff
@@ -0,0 +1,150 @@
+import SwiftUI
```

Comment:
This reminded me - attributed text is coming soon - we could use this approach until we drop iOS14 support sometime in 2021/2022 - https://betterprogramming.pub/ios-15-attributed-strings-in-swiftui-markdown-271204bec5c1 https://developer.apple.com/documentation/foundation/attributescopes/swiftuiattributes

Reply:
There was some discussion about this in paNNhX-fz-p2. That would be ideal, but it will take a while for us to be able to use that solution. I considered making the whole text a link, but there are some upcoming designs for onboarding with longer paragraphs where I don't think that behavior would feel right:

![Screen Shot 2021-07-27 at 13 47 39](https://user-images.githubusercontent.com/8739/127148613-0683720c-1752-41d1-85bc-aca222810f94.png)